dts/bindings/gpio/intel,gpio.yaml

```diff
@@ -25,6 +25,15 @@ properties:
     type: int
     description: Pin offset of this GPIO entry
 
+  acpi-ginf:
```

Comment:
Not familiar with ACPI at all, but it seems weird that a boolean describes a version. Maybe rename it to `acpi-ginf-3-param` or so?

Reply:
Will update it to `acpi-ginf-3-param`